lib/elixir/test/erlang/match_test.erl

```diff
@@ -66,6 +66,9 @@ multiple_assignment_with_binding_expression_test() ->
 underscore_assignment_test() ->
   {1, []} = eval("_ = 1").
 
+underscore_assignment_return_value_test() ->
```

Comment:
We have a kernek/comprehensions_test.exs or something similar, I would say there is the best place for a test case!